src/hooks/protections/receive_net_message.cpp

```diff
@@ -107,8 +107,10 @@ namespace big
 			case rage::eNetMessage::MsgTextMessage2:
 			{
 				char message[256];
+				rage::rlGamerHandle temp{};
+				bool is_team = false;
 				buffer.ReadString(message, 256);
-				bool is_team;
+				gamer_handle_deserialize(temp, buffer);
```

Comment:
gamer_handle_deserialize uses ReadInt64 which is incorrect. There is different logic for parsing a signed int from a bitbuffer. It should instead use ReadQWord.